pyproject.toml

```diff
@@ -31,10 +31,11 @@ exclude = [".git", "__pycache__", ".tox", "build", "dist", "*.egg_info", "venv"]
 select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore W503, E501 because using black creates errors with this
 # Ignore D107 Missing docstring in __init__
-ignore = ["W503", "E501", "D107"]
+ignore = ["W503", "E501", "D107", "DCO031"]
```

Comment:
Is this `DCO031: function/ method that does not return a value has the returns section in the docstring.`?

Reply:
I think so, but this seems to be not needed anymore, moved, thanks!